python/sparkdl/graph/builder.py

```diff
@@ -67,17 +69,101 @@ def __exit__(self, *args):
     def run(self, *args, **kargs):
         """
         This method delegate the TensorFlow graph execution
-        to the underlying tf.Session object to perform 
+        to the underlying tf.Session object to perform
         one step of graph computation.
 
         All the parameters are defined according to `tf.Session.run`
-        Reference: https://www.tensorflow.org/api_docs/python/tf/Session#run        
+        Reference: https://www.tensorflow.org/api_docs/python/tf/Session#run
         """
         return self.sess.run(*args, **kargs)
 
+    def asUDF(self, udf_name, fetches, feeds_to_fields_map=None, blocked=False, register=True):
+        """
+        Register the graph in this session as a Spark SQL UserDefinedFunction.
+
+        The following example creates a UDF that takes the input
+        from a DataFrame column named 'image_col' and produce some random prediction.
+
+        .. code-block:: python
+
+            import sparkdl.graph.utils as tfx
+
+            with IsolatedSession(using_keras=True) as issn:
```

Comment:
I had forgotten that this is just for tensorflow. The example here should be minimal, like the test below:

```python
		 +        with IsolatedSession() as issn:
 +            # The placeholder that corresponds to column 'x' as a whole column
 +            x = tf.placeholder(tf.double, shape=[], name="x")
 +            # The output that adds 3 to x
 +            z = tf.add(x, 3, name='z')
 +            # Let's register these computations in SQL.
 +            issn.asUDF("map_rows_sql_1", [z])
```